amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/DynamoUtils.java

```diff
@@ -36,6 +37,14 @@ public static AttributeValue createAttributeValue(Collection<String> collectionV
         return AttributeValue.builder().ss(collectionValue).build();
     }
 
+    public static AttributeValue createAttributeValue(byte[] byteBufferValue) {
+        if (byteBufferValue == null) {
+            throw new IllegalArgumentException("Byte buffer attributeValues cannot be null or empty.");
+        }
```

Comment:
use Validate.nonNull instead

Reply:
I like this for readability but this will be inconsistent with other `createAttributeValue` methods, which all throw `IllegalArgumentException` instead of NPE.

Reply:
Apache Validate throws IllegalArgumentException http://commons.apache.org/proper/commons-lang/javadocs/api-2.4/org/apache/commons/lang/Validate.html Is it not possible to use that?

Reply:
I don't see an issue with it, but for consistency I'm preferring this with the rest of this class which isn't using Validate